Content.Shared/Mindshield/FakeMindShield/SharedFakeMindShieldImplantSystem.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Content.Shared.Actions;
+using Content.Shared.Implants;
+using Content.Shared.Implants.Components;
+using Content.Shared.Mindshield.Components;
+
+namespace Content.Shared.Mindshield.FakeMindShield;
+
+public sealed class SharedFakeMindShieldImplantSystem : EntitySystem
+{
+    [Dependency] private readonly SharedActionsSystem _actionsSystem = default!;
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<SubdermalImplantComponent, FakeMindShieldToggleEvent>(OnFakeMindShieldToggle);
+        SubscribeLocalEvent<FakeMindShieldImplantComponent, ImplantImplantedEvent>(ImplantCheck);
+    }
+    /// <summary>
+    /// Raise the Action of a Implanted user toggling their implant to the FakeMindshieldComponent on their entity
+    /// </summary>
+    private void OnFakeMindShieldToggle(EntityUid uid,
+        SubdermalImplantComponent component,
+        FakeMindShieldToggleEvent ev)
+    {
+        ev.Handled = true;
+        if (component.ImplantedEntity is not { } ent)
+            return;
+
+        if (!TryComp<FakeMindShieldComponent>(ent, out var comp))
+            return;
+        _actionsSystem.SetToggled(ev.Action, !comp.IsEnabled); // Set it to what the Mindshield component WILL be after this
+        RaiseLocalEvent(ent, ev);
```

Comment:
Probably best to add a comment why it is an invent rather than a direct toggle, i.e. 
`// This raises an event to support an eventual future Changeling antag, who will need to interact with the FakeMindshieldComponent`